previewer/presentationvideo.js

```diff
@@ -1,3 +1,10 @@
+// add a function to skip to certain point
+function video_jump(seconds) {
```

Comment:
I would definitely put this inside the closure below.

Reply:
What should be where in javascript is very confusing for me. When I put this inside it didn't seem to be available as a a function to the divs. Should it come before or after the divs? Does it need to be injected into the page?

Reply:
If I place this within the closure the onclick events can't call it, do I need to do something to make it globally available?

Reply:
Ok, solved by a little googling